astro/src/content/blog/i-just-want-authentication-to-work.mdx

```diff
@@ -0,0 +1,77 @@
+---
+publish_date: 2023-11-17
+title: I Just Want Authentication To Work
+description: When I am writing an Application the last thing that I care about is setting up Authentication, I just want it to work!
```

Comment:
I haven't seen capital A application or authentication in the blog. Typically, when we use that in doc, we mean the Application configuration/object in FusionAuth. Is that intentional?